okhttp/src/main/kotlin/okhttp3/internal/cache/DiskLruCache.kt

```diff
@@ -613,7 +640,9 @@ class DiskLruCache(
     entry.currentEditor?.detach() // Prevent the edit from completing normally.
 
     for (i in 0 until valueCount) {
-      fileSystem.delete(entry.cleanFiles[i])
+      if (fileSystem.exists(entry.cleanFiles[i])) {
+        fileSystem.delete(entry.cleanFiles[i])
```

Comment:
I think you want `deleteIfExists` that caches a `NoSuchFileException`

Reply:
Is that a planned helper method in okio?